.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "recommendations": [
+    "nrwl.angular-console",
+    "ms-vscode.vscode-typescript-tslint-plugin",
+    "esbenp.prettier-vscode",
+    "angular.ng-template",
+    "firsttris.vscode-jest-runner"
+  ]
+}
```

Comment:
Is this strictly related to this PR?

Otherwise I suggest creating a Housekeeping issue. I have a few things also related to `.vscode/` and `.gitignore` that could go into that issue 😊

Reply:
We'll leave it in and finetune it more later on...